ee/frontend/mobile-replay/transformer/transformers.ts

```diff
@@ -310,13 +310,14 @@ export function dataURIOrPNG(src: string): string {
 }
 
 function makeImageElement(
-    wireframe: wireframeImage,
+    wireframe: wireframeImage | wireframeScreenshot,
```

Comment:
we don't currently carry the context of the parent when rendering the wireframes so can't enforce that the screenshot is being attached to the body

but we should be fine to let convention rule here